ruby/vampires.rb

```diff
@@ -0,0 +1,98 @@
+#Driver code
+#Process multiple employees
+puts "How many employees will be processed?"
+ee_num = gets.chomp.to_i
```

Comment:
While it makes sense at the time for short-hand abbreviations, when we revisit our code in the future (or someone else), such decisions may not be so clear.  Verbose is fine maybe something like `total_employees`